it/controllers/correctReturn/PackAtBusinessAddressControllerISpec.scala

```diff
@@ -175,15 +177,17 @@ class PackAtBusinessAddressControllerISpec extends ControllerITTestHelper {
                   client, correctReturnBaseUrl + normalRoutePath, Json.obj("value" -> yesSelected.toString)
                 )
 
-                whenReady(result) { res =>
-                  res.status mustBe 303
-                  res.header(HeaderNames.LOCATION) mustBe Some(defaultCall.url)
-                  val updatedAnswers = getAnswers(userAnswers.id)
-                  val dataStoredForPage = updatedAnswers.fold[Option[Boolean]](None)(_.get(PackAtBusinessAddressPage))
-                  dataStoredForPage.nonEmpty mustBe true
-                  dataStoredForPage.get mustBe yesSelected
-                  updatedAnswers.map(data => data.packagingSiteList) mustBe
+                if(yesSelected){
```

Comment:
Need an equivalent test for when no is selected - currently it's not being tested